awx/settings/defaults.py

```diff
@@ -194,6 +194,11 @@ def IS_TESTING(argv=None):
 # events into the database
 JOB_EVENT_WORKERS = 4
 
+# Maximum number of processes the callback receiver or task dispatcher will spawn
```

Comment:
This is a good idea, and is a reasonable default.  Technically the `callback_receiver` bit is wrong (it uses `JOB_EVENT_WORKERS`, and the number of worker processes stays static for the lifetime of that callback receiver).